src/drawio/modulated.js

```diff
@@ -90857,9 +90857,9 @@ EditorUi.compactUi = true;
 EditorUi.prototype.splitSize = (mxClient.IS_TOUCH || mxClient.IS_POINTER) ? 12 : 8;
 
 /**
- * Specifies the height of the menubar. Default is 34.
+ * Specifies the height of the menubar. Default is 34. MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
```

Comment:
Looks like this could be removed?

```suggestion
 * Specifies the height of the menubar. Default is 34.
```